hal/halx86/acpi/halacpi.c

```diff
@@ -1112,4 +1295,114 @@ HalReportResourceUsage(VOID)
     HalpRegisterPciDebuggingDeviceInfo();
 }
 
+#ifdef _M_IX86
+VOID
+NTAPI
+HalAcpiTimerCarry(VOID)
+{
+    ULONG Time;
+    LARGE_INTEGER Value;
+
+    Time = READ_PORT_ULONG(TimerInfo.TimerPort);
+
+    Value.QuadPart = TimerInfo.AcpiTimeValue.QuadPart + TimerInfo.ValueExt;
+    Value.QuadPart += ((Value.LowPart ^ Time) & TimerInfo.ValueExt);
+
+    TimerInfo.TimerCarry = Value.HighPart;
+    TimerInfo.AcpiTimeValue.QuadPart = Value.QuadPart;
+}
+
+VOID
+NTAPI
+HalAcpiBrokenPiix4TimerCarry(VOID)
```

Comment:
This is necessary for some chipsets. In fact, this is a black list in the registry (HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control\HAL\...). 
![Control_HAL](https://user-images.githubusercontent.com/17290914/111247777-87b77000-8600-11eb-9da9-10a774d9be9e.png)
This variable is set in Halppiix4Detect() if Bit0 is set to the registry value for this chipset.
Since to work with HW, first of all, need to change the ACPI driver, I did not add Halppiix4Detect() (and a few more subfunctions).